variables.tf

```
@@ -102,6 +102,20 @@ variable "ipv6_enabled" {
   nullable    = false
 }
 
+variable "enable_private_ipv6" {
```

```suggestion
variable "private_ipv6_enabled" {
```
I hate to be nit-picky, but is there any way you could update the name to this to keep the naming convention?